spec/openapi.yaml

```diff
@@ -1138,6 +1157,46 @@ components:
                 description: Path to package
                 example: "jo/packages"
 
+    Job:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 42
+        name:
+          type: string
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        display_path:
+          type: string
+          example: "/PrusaLink gcodes/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
```

Comment:
I think I was protesting against the `display_path` being hard to construct on my side and we had some kind of idea what to do about it? @ondratu? Is that thing really necessary?

Reply:
I have replace it by display_name, my bad I suppose :)